internal/controller/kubecertagent/execer.go

```diff
@@ -98,7 +98,8 @@ func (c *execerController) Sync(ctx controllerlib.Context) error {
 		return nil
 	}
 
-	certPath, keyPath := c.getKeypairFilePaths(maybeAgentPod)
+	certPath := getPodEnv(maybeAgentPod, "CERT_PATH")
```

Comment:
Could you see if doing `cat ${env_var}` in the exec command works?

Reply:
I tried this and it does work. We can also use a single `sh -c 'cat ${CERT_PATH}; echo; echo; cat ${KEY_PATH}'` command and easily do this in a single exec call. I think I'm going to leave this as-is until the switch to distroless (#448) because that will require some similar changes anyway.